rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -183,6 +184,44 @@ def _decode_borrow(
                 # Set protocol for both events
                 event.counterparty = CPT_AAVE_V2
 
+    def _decode_liquidation(
+            self,
+            tx_log: EvmTxReceiptLog,
+            decoded_events: list['EvmEvent'],
+    ) -> None:
+        """
+        Decode AAVE v2 liquidations. When a liquidation happens the user returns the debt token
+        and part of the collateral deposited is lost too. Those two events happen as transfers in
+        a transaction started by the liquidator.
+        """
+        if self.base.is_tracked(hex_or_bytes_to_address(tx_log.topics[3])) is False:
```

Comment:
Not sure what's better, but just a thought, that readability could be increased here by doing

```python
user_address = hex_or_bytes_to_address(tx_log.topics[3])
returned_atoken_raw_amount = hex_or_bytes_to_int(tx_log.data[32:64])
```
and then
```python
if self.base.is_tracked(user_address) is False:
```

and same with all other parts of the log.

Reply:
the user address is not used in other places and for the amounts I had them in variables but is storing values that you might not need and is used only once. the comment I added next to it is good enough